src/api/app/controllers/webui/monitor_controller.rb

```diff
@@ -90,13 +90,14 @@ def events
     check_ajax
     data = {}
 
-    arch = params[:arch] || ''
+    arch = Architecture.find_by(name: params[:arch]) || Architecture.find_by(name: 'x86_64')
+
     range = params[:range]
     ['waiting', 'blocked', 'squeue_high', 'squeue_med'].each do |prefix|
-      data[prefix] = gethistory(prefix + '_' + arch, range, !discard_cache?).map { |time, value| [time * 1000, value] }
+      data[prefix] = gethistory(prefix + '_' + arch.name, range, !discard_cache?).map { |time, value| [time * 1000, value] }
```

Comment:
Not really needed, but I like that it's clearer now :+1:, however could we use String interpolation?

Reply:
i think yes. lets see